articles/active-directory/develop/scenario-web-api-call-api-app-configuration.md

```diff
@@ -81,27 +81,19 @@ Instead of a client secret, you can provide a client certificate. The following
 
 Microsoft.Identity.Web provides several ways to describe certificates, both by configuration or by code. For details, see [Microsoft.Identity.Web wiki - Using certificates](https://github.com/AzureAD/microsoft-identity-web/wiki/Using-certificates) on GitHub.
 
-## Startup.cs
+## Program.cs
 
-Your web API will need to acquire a token for the downstream API. You specify it by adding the `.EnableTokenAcquisitionToCallDownstreamApi()` line after `.AddMicrosoftIdentityWebApi(Configuration)`. This line exposes the `ITokenAcquisition` service, that you can use in your controller/pages actions. However, as you'll see in the next two bullet points, you can do even simpler. You'll also need to choose a token cache implementation, for example `.AddInMemoryTokenCaches()`, in *Startup.cs*:
```

Comment:
I would add a note, that if the code is ASP.NET Core 3.1, or .NET 5.0, it will be the same, but located in Startup.cs: